packages/client/src/users/postPhoneNumberValidations.ts

```diff
@@ -0,0 +1,29 @@
+import client, { adaptError } from '../helpers/client';
+import type { PostPhoneNumberValidations } from './types';
+
+/**
+ * Method responsible for sending a phone token to the specified phone number.
+ *
+ * @function postPhoneNumberValidations
+ * @memberof module:profile/client
+ *
+ * @param {object} [data] - Object containing the phone number.
```

Comment:
if data is a mandatory object, it shouldn't have [].
The same applies for the rest of the files 😀